vespa-athenz/src/main/java/com/yahoo/vespa/athenz/aws/AwsCredentials.java

```diff
@@ -27,43 +27,49 @@ public class AwsCredentials implements AutoCloseable {
     private final ZtsClient ztsClient;
     private final String externalId;
     private final boolean close;
+    private final Duration duration;
```

Comment:
It's not obvious from the name of this field, nor its type, that this can legitimately be null.  Consider fixing this.